app/controllers/katello/api/v2/api_controller.rb

```diff
@@ -38,6 +36,7 @@ def resource_class
       @resource_class ||= "Katello::#{resource_name.classify}".constantize
     end
 
+    # rubocop:disable MethodLength
```

Comment:
This replaces 2 lines with 5 lines. Previously it was 27 lines (which was under the 30 line limit). Now it's 32 which is greater than the MethodLength limit.